kinematics_interface/include/kinematics_interface/kinematics_interface.hpp

```diff
@@ -39,10 +39,15 @@ class KinematicsInterface
 
   /**
    * \brief Initialize plugin. This method must be called before any other.
+   * \param[in] robot_description robot URDF in string format
+   * \param[in] parameters_interface 
+   * \param[in] param_namespace namespace for kinematics parameters - defaults to "kinematics"
+   * \return true if successful
    */
   virtual bool initialize(
+    const std::string & robot_description,
     std::shared_ptr<rclcpp::node_interfaces::NodeParametersInterface> parameters_interface,
-    const std::string & end_effector_name) = 0;
+    const std::string & param_namespace = "kinematics") = 0;
```

Comment:
Do we want to hardcode this here?

Reply:
You are right, we don't need this here:

```suggestion
    const std::string & param_namespace) = 0;
```